keps/sig-node/4369-allow-special-characters-environment-variable/README.md

```diff
@@ -224,7 +224,9 @@ Yes.
 
 ### Rollout, Upgrade and Rollback Planning
 
-When the feature gate is disabled, workloads that are already running will not be affected. If environment variables contain special characters, changes to fields other than the environment variables will not cause workloads to fail. However, if the environment variable fields are modified, they may fail to recreate Pods or ReplicaSets due to the Apiserver's validation logic, which could result in workload failures.
+###### How can a rollout or rollback fail? Can it impact already running workloads?
```

Comment:
I reverted it to the previous description of https://github.com/kubernetes/enhancements/pull/4805#discussion_r1766894555